core/src/main/java/nl/nn/adapterframework/pipes/IteratingPipe.java

```diff
@@ -422,10 +422,10 @@ public void waitForResults() throws SenderException, IOException {
 							if (senderResult.isSuccess()) {
 								itemResult = senderResult.getResult().asString();
 							} else {
-								itemResult = "<exception>"+ XmlEncodingUtils.encodeChars(senderResult.getResult().asString())+"</exception>";
```

Comment:
You can't just change this behavior. There are many applications using this which will break if we merge this as is.

Please make this behavior configurable, with an attribute for example `throwExceptions="true"`.

Also don't forget to make a few tests for this. To test both the old and new behavior 😃.